google/cloud/bigquery/v2/minimal/internal/job_request.cc

```diff
@@ -70,6 +71,25 @@ StateFilter StateFilter::Done() {
   return state_filter;
 }
 
+std::string GetJobRequest::DebugString(TracingOptions const& options) const {
+  std::string out;
+  absl::StrAppend(&out, "GetJobRequest{project_id=", project_id_,
+                  ", job_id=", job_id_, ", location=", location_, "}");
+  return internal::DebugString(out, options);
+}
+
+std::string ListJobsRequest::DebugString(TracingOptions const& options) const {
+  std::string out;
+  std::string all_users = all_users_ ? "true" : "false";
+  absl::StrAppend(&out, "ListJobsRequest{project_id=", project_id_,
+                  ", all_users=", all_users, ", max_results=", max_results_,
+                  ", page_token=", page_token_,
+                  ", projection=", projection_.value,
+                  ", state_filter=", state_filter_.value,
+                  ", parent_job_id=", parent_job_id_, "}");
+  return internal::DebugString(out, options);
```

Comment:
same goes for this one. we need
- a custom delimeter that depends on `options`
- individual string fields to use `DebugString(..., options)`